curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-052.md

```diff
@@ -7,16 +7,16 @@ dashedName: part-52
 
 # --description--
 
-Something does not look right. You added the correct `class` attribute value to the `p` element with `Donunt` as its text, but you have not defined a selector for it.
+Something does not look right. You added the correct `class` attribute value to the `p` element with `Donut` as its text, but you have not defined a selector for it.
 
 Since the `flavor` class selector already has the properties you want, just add the `dessert` class name to it.
 
 # --hints--
 
-Test 1
+You should add the `.dessert` selector to your `.flavor` selector. 
 
 ```js
-
+assert(code.match(/\.(flavor,\s*.dessert)|(dessert,\s*.flavor)\s*{/i));
```

Comment:
```suggestion
assert(__helpers.removeWhiteSpace(code).match(/\.(flavor,.dessert)|(dessert,.flavor){/i));
```
You might be asking what the difference is, but this allows: `.flavor , dessert{`

Reply:
I am always skeptical of that `removeWhiteSpace` function - wouldn't it allow users to pass with any spaces where they shouldn't be? `.fl  avor, .des sert`. Also, these regex's are missing the espace character in front of the period I think.

Reply:
> wouldn't it allow users to pass with any spaces where they shouldn't be? `. fl avor, .des sert`

Honestly, I had never even considered these cases 😨